File: Java/src/World/body/plant/Belladonna.java
package World.body.plant;

import GUI.Color_obj;
import World.Point;
import World.World;
import World.body.Plant;

import java.io.Serializable;

public class Belladonna extends Plant implements Serializable {
    private static final long serialVersionUID = 999999999L;
    public Belladonna(Point point_location, World world) {
        super(point_location, world);
        setName("Belladonna");
        setPower(999);
        setColor(Color_obj.BELLADONNA);
    }

    @Override
    public int get_stronger() {
        return 0;
    }
}

File: Python/World/Body/Plant/belladonna.py
from World.Body.plant import Plant
from World.point import Point
from GUI.colors import Colors


class Belladonna(Plant):
    def __init__(self, point_location, world):
        super().__init__(point_location, world)
        self.set_name("Belladonna")
        self.set_power(999)
        self.set_color(Colors.BELLADONNA)

    def get_stronger(self):
        return 0
